.github/CODEOWNERS

```diff
@@ -19,6 +19,7 @@ packages/azure-theme/ @microsoft/azure-ux-design-engineering
 packages/variant-theme @microsoft/fluentui-variant-theme
 packages/react-keytips @microsoft/cxe-prg @mainframev
 packages/react-interactive-tab @microsoft/cxe-prg @dmytrokirpa
+packages/react-gamepad-navigation @microsoft/cxe-prg @hectorjjb
```

Comment:
can you update your github profile to verify your origin ? based on current state we are unable to understand where is this coming from. ty